tools.go

```diff
@@ -0,0 +1,8 @@
+// +build tools
+
+// This file just exists to ensure we download the tools we need for building
+// See https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module
+
+package helm_operator
+
+import "github.com/fluxcd/helm-operator/pkg/install"
```

Comment:
Newline

Reply:
I went with the import format used in Flux